### PR TITLE
added the sendCommand function

### PR DESCRIPTION
**Pull request by makermelissa**
I created a non-const and const version as we discussed and also an overloaded one that allows just the command byte for functions like the SSD1351 invert, which I tested.

### CODE REVIEW

**Comment by makermelissa** [comment]
I'm not sure what's going on with Travis for the Zero, but it compiled in travis on my repo and also compiles fine on a computer.

**Comment by makermelissa** [comment]
Ok, looks like it had a bad Zero Board package cached. Temporarily commenting out in .travis.yml, committing, then uncommenting and committing seems to have forced it to fix itself.

**Comment by ladyada** [comment]
you can delete the cache in travis - please do that instead of changing travis.yml :)

**Comment by makermelissa** [comment]
Oh sweet. I didn't realize that. Thanks!

File: Adafruit_SPITFT.cpp [hunk 1]
```diff
@@ -1632,6 +1632,51 @@ uint16_t Adafruit_SPITFT::color565(uint8_t red, uint8_t green, uint8_t blue) {
...
 
+/*!
+ @brief   Adafruit_SPITFT Send Command handles complete sending of commands and data
+ @param   commandByte       The Command Byte
+ @param   dataBytes         A pointer to the Data bytes to send
+ @param   numDataBytes      The number of bytes we should send
+ */
+void Adafruit_SPITFT::sendCommand(uint8_t commandByte, uint8_t *dataBytes, uint8_t numDataBytes) {
+    SPI_BEGIN_TRANSACTION();
+    if(_cs >= 0) SPI_CS_LOW();
```
Comment (ladyada) [inline Adafruit_SPITFT.cpp]: change to _cs > 0, as pin #0 is a valid pin

File: Adafruit_SPITFT.cpp [hunk 2]
```diff
@@ -1632,6 +1632,51 @@ uint16_t Adafruit_SPITFT::color565(uint8_t red, uint8_t green, uint8_t blue) {
...
+    SPI_DC_LOW(); // Command mode
+    spiWrite(commandByte); // Send the command byte
+  
+    SPI_DC_HIGH();
+    for (int i=0; i<numDataBytes; i++) {
+      spiWrite(*dataBytes); // Send the data bytes
+      dataBytes++;
+    }
+  
+    if(_cs >= 0) SPI_CS_HIGH();
```
Comment (ladyada) [inline Adafruit_SPITFT.cpp]: ditto

File: Adafruit_SPITFT.cpp [hunk 3]
```diff
@@ -1632,6 +1632,51 @@ uint16_t Adafruit_SPITFT::color565(uint8_t red, uint8_t green, uint8_t blue) {
...
+
+/*!
+ @brief   Adafruit_SPITFT Send Command handles complete sending of commands and const data
+ @param   commandByte       The Command Byte
+ @param   dataBytes         A pointer to the Data bytes to send
+ @param   numDataBytes      The number of bytes we should send
+ */
+void Adafruit_SPITFT::sendCommand(uint8_t commandByte, const uint8_t *dataBytes, uint8_t numDataBytes) {
+    SPI_BEGIN_TRANSACTION();
+    if(_cs >= 0) SPI_CS_LOW();
```
Comment (ladyada) [inline Adafruit_SPITFT.cpp]: ditto

File: Adafruit_SPITFT.cpp [hunk 4]
```diff
@@ -1632,6 +1632,51 @@ uint16_t Adafruit_SPITFT::color565(uint8_t red, uint8_t green, uint8_t blue) {
...
+  
+    SPI_DC_LOW(); // Command mode
+    spiWrite(commandByte); // Send the command byte
+  
+    SPI_DC_HIGH();
+    for (int i=0; i<numDataBytes; i++) {
+      spiWrite(pgm_read_byte(dataBytes++)); // Send the data bytes
+    }
+  
+    if(_cs >= 0) SPI_CS_HIGH();
```
Comment (ladyada) [inline Adafruit_SPITFT.cpp]: ditto

Comment (makermelissa) [reply Adafruit_SPITFT.cpp]: 👍

File: Adafruit_SPITFT.cpp (hunk 3 above)
Comment (makermelissa) [reply Adafruit_SPITFT.cpp]: 👍

File: Adafruit_SPITFT.cpp (hunk 2 above)
Comment (makermelissa) [reply Adafruit_SPITFT.cpp]: 👍

File: Adafruit_SPITFT.cpp (hunk 1 above)
Comment (makermelissa) [reply Adafruit_SPITFT.cpp]: 👍

**Pull request merged**